about.html

```diff
@@ -0,0 +1,240 @@
+                    experts and thought leaders, who share their knowledge and experience with the attendees. Many
+                    developer conferences also feature exhibits and demos of new products and tools, giving attendees
+                    the opportunity to try out the latest technology firsthand.</p>
+            </div>
+        </div>
+
+        <div class="email_contact_wrapper">
+            <div class="email_contact_container">
+                <p class="text_to_email">Please contact us per Email for nay further questions about CC Global Summit
+                    2015!</p>
+                <a href="#" class="email_contact">ccoderssum2023@cccoders.org ></a>
+            </div>
+        </div>
+    </section>
+
+    <section class="summit_info_wrapper">
+        <div class="summit_info">
+            <div class="summit_title_wrapper">
+                <div class="summit_title_container">
+                    <h2 class="summit_title">Creative Coders <br class="hide--desktop">
+                        Global Summit 2015 Logo</h2>
+                    <hr class="summit_bar">
+                </div>
+            </div>
+            <div class="summit_description_wrapper">
+                <div class="summit_description">
+                    <p class="hide--desktop">The logo of CC Global Summit 2015 was decided through the logo competition
+                        from 8, June to 7,
+                        July.
+                    </p>
+                    <p class="hide--mobile">
+                        The logo of CC Global Summit 2015 was decided through the logo competition from 8, June to 7,
+                        July.
+                        After online public voting and Selection Committee’s judging process, the logo from Naresh
+                        Agrawal won the prize.
+                        After consultation with Mr. Agrawal, we optimized the prize-winning-logo for this year’s summit
+                        as follows:
+                    </p>
+                    <div class="summit_logo">
+                        <img class="logo_summit_desktop" src="./assets/images/logo.png" alt="about-logo">
+                    </div>
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <section class="past_summit_wrapper">
+        <div class="past_summit">
+            <div class="past_summit_title_wrapper">
+                <div class="past_summit_title_container">
+                    <h2 class="past_summit_title">See the past CC Global Summits</h2>
+                    <hr class="past_summit_bar">
+                </div>
+            </div>
+            <div class="past_summit_description_wrapper">
+                <div class="past_summit_description">
+                    <p>
+                        Take a look at the past two CC Global Summits which took place in Buenos Aires and in Warsaw
+                    </p>
+                </div>
+            </div>
```

Comment:
- [ ] Did you mean Global Summit 2023? Please 🙏🏽 correct the year and the information about your logo **CC Global Summits** means **Creative Commons Global Summits** to personalize your page according to the topic.